elyra/metadata/schemas/runtime-image.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Schema for Runtime Image metadata",
+  "name": "runtime-image",
+  "properties": {
+    "schema_name": {
+      "type": "string",
+      "pattern": "^[a-z][a-z0-9-_]*[a-z0-9]$",
+      "minLength": 1
+    },
+    "display_name": {
+      "description": "The display name of the Runtime Image",
+      "type": "string",
+      "pattern": "^[a-zA-Z][a-zA-Z0-9-_. (){}]*[a-zA-Z0-9)}]$"
+    },
+    "metadata": {
+      "description": "Additional data specific to this Runtime Image",
+      "type": "object",
+      "properties": {
+        "image_name": {
+          "description": "Runtime Image description",
```

Comment:
Does this include the tag?  If so, we should probably mention that in the description.

Reply:
Per #497 - we should add an `image_tag` property, and make it required with a default value of 'latest'.

Reply:
that will come when we implement #497